crates/notion-core/src/inventory/serial.rs

```diff
@@ -108,6 +108,7 @@ pub struct NodeEntry {
     #[serde(with = "option_version_serde")]
     pub npm: Option<Version>,
     pub files: Vec<String>,
+    pub lts: bool,
```

Comment:
It looks like the issue with the test is here: I didn't look as closely as I should have at the `index.json` for Node. It looks like the `lts` property in the Node index is actually of type `false | string`. If it's false, that means no LTS, if it's a string value (like "Dubnium"), then it's that named LTS group (Node 10 is `Dubnium`, Node 8 is `Carbon`, etc.)

It looks like we'll need to have a custom deserializer for that field, that can convert a string to a boolean value. You can take a look at [this block](https://github.com/notion-cli/notion/blob/master/crates/notion-core/src/version/mod.rs#L152) for implementing a custom deserializer. In our case, we probably want to try to deserialize the value as a string, and if that succeeds we return `Ok(true)`, otherwise `Ok(false)`. Then we can annotate this line with `#[serde(deserialize_with = "<NAME OF FUNCTION>")]`, and it should be able to parse it the way we want.

Reply:
I've gone ahead and implemented my best attempt at this, please let me know what you think!